sktime/transformations/series/kalman_filter.py

```diff
@@ -760,8 +760,25 @@ def get_test_params(cls, parameter_set="default"):
             instance.
             ``create_test_instance`` uses the first (or only) dictionary in ``params``
         """
-        params = {"state_dim": 2}
-        return params
+        params1 = {"state_dim": 2}
+        params2 = {
+            "state_dim": 2,
+            "initial_state": np.array([0, 0]),
+            "initial_state_covariance": np.array([[0.1, 0], [0.1, 0]]),
+            "state_transition": np.array([[1, 0.1], [0, 1]]),
+            "process_noise": np.array(
+                [
+                    [1 / 4 * (0.1**4), 1 / 2 * (0.1**3)],
+                    [1 / 2 * (0.1**3), 0.1**2],
+                ]
+            )
+            * 0.1,
+            "measurement_function": np.array([[1, 0]]),
```

Comment:
I have to comment out `measurement_function` and `measurement_noise` to get `KalmanFilterTransformerPK` to pass the `check_estimator(kf.KalmanFilterTransformerPK, raise_exceptions=True)` test, but I don't think it's a good idea to simply delete the two parameters, could you @fkiraly help me to resolve this?
```python
from sktime.transformations.series import kalman_filter as kf
from sktime.utils.estimator_checks import check_estimator


check_estimator(kf.KalmanFilterTransformerFP)
# All tests PASSED!
check_estimator(kf.KalmanFilterTransformerPK, raise_exceptions=True)
# Traceback (most recent call last):
#   File "/home/xinyu/Documents/sktime/sktime-dev/temp/test_KalmanFilterTransformer.py", line 7, in <module>
#     check_estimator(kf.KalmanFilterTransformerPK, raise_exceptions=True)
#   File "/home/xinyu/Documents/sktime/sktime/utils/estimator_checks.py", line 116, in check_estimator
#     test_cls_results = test_cls().run_tests(
#                        ^^^^^^^^^^^^^^^^^^^^^
#   File "/home/xinyu/Documents/sktime/sktime/tests/test_all_estimators.py", line 605, in run_tests
#     test_fun(**deepcopy(args))
#   File "/home/xinyu/Documents/sktime/sktime/tests/test_all_estimators.py", line 1230, in test_fit_does_not_overwrite_hyper_params
#     fitted_est = scenario.run(estimator_instance, method_sequence=["fit"])
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
#   File "/home/xinyu/Documents/sktime/sktime/utils/_testing/scenarios.py", line 180, in run
#     res = getattr(obj, methodname)(**args)
#           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
#   File "/home/xinyu/Documents/sktime/sktime/transformations/base.py", line 498, in fit
#     self._fit(X=X_inner, y=y_inner)
#   File "/home/xinyu/Documents/sktime/sktime/transformations/series/kalman_filter.py", line 640, in _fit
#     ) = self._em(X=X, measurement_dim=measurement_dim, state_dim=self.state_dim)
#         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
#   File "/home/xinyu/Documents/sktime/sktime/transformations/series/kalman_filter.py", line 813, in _em
#     kf = KalmanFilter(
#          ^^^^^^^^^^^^^
#   File "/home/xinyu/Documents/sktime/env/lib/python3.12/site-packages/pykalman/standard.py", line 1029, in __init__
#     n_dim_obs = _determine_dimensionality(
#                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
#   File "/home/xinyu/Documents/sktime/env/lib/python3.12/site-packages/pykalman/standard.py", line 75, in _determine_dimensionality
#     raise ValueError(
# ValueError: The shape of all parameters is not consistent.  Please re-check their values.
```